res/css/views/rooms/_EventTile.pcss

```diff
@@ -449,6 +454,16 @@ $left-gutter: 64px;
     }
 
     &[data-layout="bubble"] {
+        /* Other half of the gutter is provided by margin-bottom on the last tile
+        of the section */
+        margin-top: calc(var(--gutterSize) / 2);
+        margin-inline-start: var(--EventTile_bubble-margin-inline-start);
+
+        /* Reset default values. TODO: remove */
+        max-width: unset;
+        clear: unset;
+        padding-top: 0;
```

Comment:
Because these have not been declared https://github.com/matrix-org/matrix-react-sdk/pull/9033/files#diff-bbaa61a6c13c01a902acef6523cff9d0b6b41fb8a7b480d708ed11187500b30bR42-L53 on `_EventBubbleTile.pcss`. See above.